.circleci/config.yml

```diff
@@ -511,24 +520,25 @@ jobs:
       - install_extension
       - restore_cache: &INTEGRATION_COMPOSER_CACHE_KEY
           key: 'betav1-composer-base-integration-deps-{{ .Environment.CIRCLE_JOB }}-{{ checksum "composer.json" }}'
-      - run: composer install --no-interaction --prefer-dist
+      - <<: *STEP_COMPOSER_UPDATE
+      - <<: *STEP_COMPOSER_TESTS_UPDATE
       - <<: *STEP_PREPARE_TEST_RESULTS_DIR
       - <<: *STEP_EXPORT_CI_ENV
       - <<: *STEP_WAIT_AGENT
       - <<: *STEP_WAIT_MYSQL
       - <<: *STEP_WAIT_REQUEST_REPLAYER
       - run:
           name: Run auto-instrumentation tests
-          command: composer test-auto-instrumentation -- --log-junit test-results/php-auto-instrumentation/results.xml
+          command: make test_auto_instrumentation PHPUNIT_OPTS="--log-junit test-results/php-auto-instrumentation/results.xml"
```

Comment:
Can we put `PHPUNIT_OPTS` in the `environment:` section so we don't repeat it in each command?

Reply:
I am afraid not, unless we come up with any good ideas. If you take this specific example that you commented, you have for the same job 4 commands (lines 532, 535, 538, and 541) that requires different php unit options (specifically for the test results file path).

It might be possible if we split more tests into separate jobs, though.

Reply:
@morrisonlevi are you 👍  with my answer to this comment?

Reply:
👍